### ISSUE TITLE
Recursive snapshots don't work

### ISSUE THREAD

**Issue by Luflosi** [khenderick/zfs-snap-manager#36]
I can't get recursive snapshots to work. This the content of `/etc/zfssnapmanager.cfg`:
```conf
[tank]
mountpoint = /mnt/storage
time = 04:00
snapshot = True
schema = 7d3w11m1y
recursive = True
```
Even though I added `recursive = True`, the snapshots are only created on `tank`. If I manually create a snapshot with `sudo zfs snapshot -r tank@recursive-test`, it works as expected.
Do you have an idea why this isn't working? Is there a good way for me to debug this?

**Comment by auberginepop** [comment]
+1 for this. If it worked it would be a great feature.

**Comment by ChrisJAllan** [comment]
Taking recursive snapshots should work in the latest git, but not the latest release. Deleting recursive snapshots, however, does not. I wrote the fix for that a while ago, and then forgot to make a PR until now.

**Comment by khenderick** [comment]
Hmm, seems I never released that. Let's do this now.

**Comment by khenderick** [comment]
Released 0.3.0 here and on AUR. It should include all latest changes. Let me know if you'd have any troubles.
